modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,252 @@
+
+Current ingress support for Mule apps on the Runtime Fabric platform is confined to https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes ingress] resources, which restrict you from using cloud vendor-native HTTP routing solutions, such as OpenShift Routes on OpenShift clusters, Istio, or Service Mesh. To address this limitation, Runtime Fabric supports a custom resource `httproutetemplates.rtf.mulesoft.com` template.
+
+Use the HTTP route-template custom resource in Runtime Fabric to define and implement custom routing configurations. With this approach, you can take advantage of compatibility with a range of ingress controllers and cloud-native routing mechanisms, including support for:
+
+* Wildcard for host-based routing
+* Map a set of application endpoints to arbitrary HTTP routing resources for path-based routing
+* Synchronization of domains or hosts from the HTTP route templates with the Anypoint control plane
+* Using parameter placeholders in `httproutetemplates.rtf.mulesoft.com` templates for additional extensibility
+
+== HTTP Route-Template Custom-Resource Placeholders
+
+Runtime Fabric provides placeholder properties that cluster administrators can use in their HTTP route-template custom resource. These placeholders are evaluated and replaced with actual data by Runtime Fabric when rendering the deployment charts.
+
+As a cluster administrator, use these HTTP route-template placeholder properties in your CR (custom resource):
+
+[%header%autowidth.spread]
+|===
+| Property| Description
+| `{{ .ResourceName }}`| Runtime Fabric generated unique routing resource name
+| `{{ .ApplicationName }}`| Mule app deployment name
+| `{{ .Namespace }}`| Namespace of the Mule app deployment
+| `{{ .Service.Name }}`| Backend service name of the Mule app deployment
+| `{{ .Service.PortName }}`| Backend service port name of the Mule app deployment
+| `{{ .Service.Port }}`| Backend service port number of the Mule app deployment
+| `{{ .Host }}`| Host or domain value from the public endpoint of Mule  app deployment
+| `{{ .Path }}`| Path value from the public endpoint of Mule app deployment
+| `{{ .OrganizationID }}`| Organization ID of the Mule app deployment
+| `{{ .EnvironmentID }}`| Environment ID of the Mule app deployment
+| `{{ .UniqueId }}`| Unique ID placeholder to create a unique application URL.
+|===
+
+== How the HTTP Route-Template Custom Resource Works in Runtime Fabric
+
+This diagram provides an overview of how to configure ingress using an HTTP route-template custom resource in Runtime Fabric:
+
+image::ingress-http-diagram.png[HTTP route-template diagram overview]
+
+The `httproutetemplates` resource applies only to new Mule app deployments created after the template resource is set up. It doesn't affect existing deployments. If you redeploy a Mule app after creating or updating the `httproutetemplates`, the new template is applied.
```

Comment:
> It doesn't affect existing deployments. If you redeploy...
Existing deployments are unaffected. After applying an `httproutetemplate`, it is applied for all subsequent application deployments.

(I'm trying to avoid the word "you" - it is not normally seen in documentation like this IMO)